technical-reports/format/index.html

```diff
@@ -43,7 +43,7 @@
         tocIntroductory: true,
         logos: [
           {
-            src: '/_includes/assets/images/logo_128_2x.png',
+            src: '/assets/images/logo_128_2x.png',
```

Comment:
These images were moved out of `_includes` to a more standard `public/assets/` folder. In the previous setup, the behavior of `_includes` was: “it’s unclear whether or not files get published; it depends on configuration in other files and plugins.” There are also some “magic” properties of eleventy and Nunjucks here that overlap, making it further unclear.

In the new setup, it was split into 2 folders:

- `_includes`: ONLY for template building in Eleventy/Nunjucks. Nothing gets published.
- `public/`: ONLY for assets. Everything gets published.